iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -289,6 +289,23 @@ void RouDi::processMessage(const runtime::IpcMessage& message,
                            const iox::runtime::IpcMessageType& cmd,
                            const RuntimeName_t& runtimeName) noexcept
 {
+    if (runtimeName.empty())
+    {
+        IOX_LOG(ERROR, "Got message with empty runtime name!");
+        return;
+    }
+
+
+    for (const auto s : platform::IOX_PATH_SEPARATORS)
+    {
+        const char separator[2]{s};
+        if (runtimeName.find(separator).has_value())
```

Comment:
For later: Shall we create a custom type runtime name that does all validation checks for us, like the `FileName`. Would a `FileName` be always a valid runtime name, if so we could maybe us it via a type alias.

Reply:
Sure, I can create an issue or if there is already an issue for these semantic types I can add the `RuntimeName_t` as item.

Reply:
Added to #2018